surprisal/model.py

```diff
@@ -32,12 +33,22 @@ def __init__(
         model_id: str,
         model_class: typing.Callable,
         device: str = "cpu",
+        precision: str = "fp32",
+        trust_remote_code: bool = False,
     ) -> None:
         super().__init__(model_id)
-
+        precisions = {"fp32": torch.float32, "fp16": torch.float16, "bf16": torch.bfloat16}
+        if precision not in precisions:
+            raise ValueError(
+                f"precision must be one of {list(precisions.keys())}, got {precision}"
+            )
         self.tokenizer = AutoTokenizer.from_pretrained(self.model_id)
         # self.model_class = model_class
-        self.model: PreTrainedModel = model_class.from_pretrained(self.model_id)
+        self.model: PreTrainedModel = model_class.from_pretrained(
+            self.model_id, 
+            torch_dtype=precisions[precision],
+            trust_remote_code=trust_remote_code,
```

Comment:
not sure I understand what this is for. is this to load checkpoints that aren't locally stored?